packages/kbn-grid-layout/grid/grid_panel/drag_handle.tsx

```diff
@@ -101,17 +100,16 @@ export const DragHandle = React.forwardRef<
         position: absolute;
         align-items: center;
         justify-content: center;
-        top: -${euiThemeVars.euiSizeL};
-        width: ${euiThemeVars.euiSizeL};
-        height: ${euiThemeVars.euiSizeL};
-        z-index: ${euiThemeVars.euiZLevel3};
```

Comment:
@Heenawter Can you confirm if it's okay to remove this z-index from the drag handle? From what I could tell testing locally, the z-index isn't doing anything here.

Reply:
This is the **native** drag handle that is used if the panel renderer isn't providing one. To see it, remove `setDragHandles` from the `ReactEmbeddableRenderer` in `examples/grid_example/public/app.tsx`. 

Removing this z-index causes a strange edge case where the `ReactEmbeddableRenderer`'s drag handle floats over the native drag handle, which makes it hard to drag panels around - but this won't ever happen in production, so I don't see an issue with removing it 🤷